packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -476,14 +477,7 @@ function hasAnyConfiguration(provider: ProviderInfo) {
               <div class="mt-2 text-gray-700 text-xs" aria-label="{container.name} type">
                 {#if container.type === 'docker'}Docker{:else if container.type === 'podman'}Podman{/if} endpoint
               </div>
-              <div
-                class="mt-1 my-auto text-xs truncate"
-                class:text-gray-900="{container.status !== 'started'}"
-                aria-label="{container.name} endpoint"
-                title="{container.endpoint.socketPath}">
-                {container.endpoint.socketPath}
-              </div>
-
+              <PreferencesResourcesRenderingCopyButton path="{container.endpoint.socketPath}" />
```

Comment:
here we should propagate the `class:text-gray-900="{container.status !== 'started'}"` to PreferencesResourcesRenderingCopyButton

Reply:
look at

```
  class="{$$props.class}"
```
that we're using in some components